res/css/views/rooms/_EventTile.pcss

```diff
@@ -1032,77 +1032,106 @@ $left-gutter: 64px;
     --ThreadView_group_spacing-end: 8px; /* same as padding */
 
     .mx_EventTile {
-        display: flex;
-        flex-direction: column;
-
         .mx_EventTile_roomName {
             display: none;
         }
 
         /* handling for hidden events (e.g reactions) in the thread view */
         &.mx_EventTile_info {
             .mx_EventTile_avatar {
-                position: absolute;
-                top: 1.5px; /* Align with hidden event content */
-                margin-top: 0;
-                margin-bottom: 0;
+                margin-block: 0;
```

Comment:
I have to say, I find the separate `margin-top` and `margin-bottom` properties easier to reason about. Is there a particular reason to prefer `margin-block`?